cvat/apps/engine/models.py

```diff
@@ -1345,3 +1345,9 @@ class RequestSubresource(TextChoices):
     ANNOTATIONS = "annotations"
     DATASET = "dataset"
     BACKUP = "backup"
+
+
+class RedisMigration(models.Model):
+    # todo: redis_inmem/redis_ondisk
```

Comment:
What do you mean by this?

Reply:
Initially, I thought that it could be useful to store to which Redis instance a migration was applied